src/web/routes/application/terms-and-conditions/post.js

```diff
@@ -12,9 +12,9 @@ const { isErrorStatusCode } = require('./predicates')
 const { CLAIMS_ENDPOINT, NO_ELIGIBILITY_STATUS_MESSAGE } = require('./constants')
 const { render } = require('./get')
 
-const transformResponse = (body, response) => {
+const handleErrorResponse = (body, response) => {
   if (isErrorStatusCode(response.statusCode)) {
-    throw new Error('Error posting to claimant service')
+    throw new Error(JSON.stringify(body))
```

Comment:
Wouldn't this be better as `throw new Error('Error posting to claimant service: ' + JSON.stringify(body))` to retain the context of when the error occurs?

Reply:
Also, could we output the HTTP response code too, that could also be useful.

Reply:
The error is caught and wrapped with "Error posting to claimant service" on line 67. This removes a double occurrence of "Error posting to claimant service"